core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java

```diff
@@ -201,11 +199,7 @@ DefaultHttpJsonResponse doRequest(int timeout,
 
             if (hasQueryParams) {
                 for (Pair<String, ?> parameter : parameters) {
-                    String name = URLEncoder.encode(parameter.first, "UTF-8");
-                    String value = parameter.second == null ?
-                                   null :
-                                   URLEncoder.encode(String.valueOf(parameter.second), "UTF-8");
-                    ub.queryParam(name, value);
+                    ub.queryParam(parameter.first, parameter.second == null ? null : String.valueOf(parameter.second));
```

Comment:
If you put `parameter.second` to the `ub.queryParam` the behaviour will be the same as with ternary operator